update mock up

fazzabd opened the pull request:
ada beberapa yg belum beres di mock up admin, belum bisa tambah lokasi

fazzabd committed:
update mock up

ada beberapa yg belum beres di mock up admin, belum bisa tambah lokasi

MaulanaIbrahim commented:
1. kenapa pemilihan jam dibuat dropdown karena kalau dilihat dari sisi user itu kurang efisien
solusi : kenapa tidak form input type biasa.
2. untuk yang initial location itu tidak ada tempat untuk menampungnya di database
3. untuk membedakan antara initial location dan destination location bagaimana jika ingin mengambil data dari database

fazzabd commented:
[mockup.zip](https://github.com/A2JTK16/Class-dan-MockUp/files/1385402/mockup.zip)

untuk nomer 1 sudah di benarkan, namun untuk nomer 2 & 3 di database nantinya ada 2 lokasi, ada lokasi 1 dan lokasi 2, dan itupun berbeda kode, sehingga bisa dibedakan

fazzabd committed:
Add files via upload

membenarkan mock up hasil review, yaitu pemilihan waktu

fazzabd committed:
Add files via upload

update mock up, benerin dari hasil review, ganti cara pilih jam, dan rapihin yg lain juga
